doc/Python.md

```diff
@@ -42,24 +43,87 @@ is required).
 You can also use existing Halide Generators (written in either C++ or Python)
 to produce Python extensions that can be used within Python code.
 
-## Python Requirements
+## Acquiring the Python bindings
 
-Before building, you should ensure you have prerequite packages installed in
-your local Python environment. The best way to get set up is to use a virtual
-environment:
+As of Halide 19.0.0, we provide binary wheels on PyPI which include the Python
+bindings and the C++/CMake package for native development. Full releases may be
+installed with `pip` like so:
 
-```console
+```shell
+$ pip install halide
+```
+
+Every commit to `main` is published to Test PyPI as a development version and
+these may be installed with a few extra flags:
+
+```shell
+$ pip install halide --pre --extra-index-url https://test.pypi.org/simple
+```
+
+Currently, we provide wheels for: Windows x86-64, macOS x86-64, macOS arm64, and
+Linux x86-64. The Linux wheels are built for manylinux_2_28, which makes them
+broadly compatible (Debian 10, Ubuntu 18.10, Fedora 29).
+
+## Building the Python bindings
+
+If `pip` isn't enough for your purposes, or you are developing Halide directly,
+you have two options for building and using the Python bindings. Note that the
+bindings require both Halide and LLVM to be built with RTTI and exceptions
```

Comment:
Building Both Halide and LLVM with RTTI is required.

Building Halide with exceptions is required.

Building LLVM with exceptions is **not** required. (I'm not even sure if it's really possible.)

Reply:
> I'm not even sure if it's really possible.

The pip packaging stuff builds LLVM with exceptions 🙂

Reply:
Fixed the language.